public/v4/apps/appwrite.yml

```diff
@@ -0,0 +1,1338 @@
+captainVersion: 4
+
+services:
+    $$cap_appname-redis:
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+        image: 'bitnami/redis:6.0.8-debian-10-r6'
+        ports:
+            - '6379:6379'
```

Comment:
Does the port need to be exposed to the public? If it's only being used by AppWrite, then, probably it won't need to be exposed.

Reply:
I did remove the ports section.